pallets/hyperdrive-outgoing/Cargo.toml

```diff
@@ -0,0 +1,89 @@
+authors = ["Papers AG", "Parity Technologies <admin@parity.io>"]
+description = "Acurast Hyperdrive is a building block allowing for general bidirectional message passing"
+version = "0.0.1"
+license = "MIT"
+homepage = "https://docs.acurast.com/"
+edition = "2021"
+publish = false
+repository = "https://github.com/acurast/acurast-core"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
+	"derive",
+] }
+sp-core = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
+
+# Acurast
+pallet-acurast = { path = "../acurast", default-features = false }
+pallet-acurast-marketplace = { path = "../marketplace", default-features = false}
+acurast-common = { path = "../acurast/common", default-features = false }
+mmr-lib = { package = "ckb-merkle-mountain-range", version = "0.5.2", default-features = false }
+
+# SDKs
+tezos-core = { git = "https://github.com/airgap-it/tezos-rust-sdk", default-features = false, branch = "feat/nostd" }
+tezos-michelson = { git = "https://github.com/airgap-it/tezos-rust-sdk", default-features = false, branch = "feat/nostd" }
+derive_more = { version = "0.99.17", default-features = false }
+strum = { version = "0.24.1", default-features = false }
+strum_macros = { version = "0.24.1", default-features = false }
+once_cell = { version = "1.17.1", default-features = false }
+
+# Benchmarks
+hex-literal = { version = "0.3", optional = true }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", optional = true, default-features = false }
+
+# Substrate
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+scale-info = { version = "2.2.0", default-features = false, features = [
+	"derive",
+] }
+sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+sp-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
+
+
+serde = { version = "1.0.136", features = ["derive"], optional = true }
+thiserror = "1.0"
+log = { version = "0.4.17", default-features = false }
+
+[dev-dependencies]
+hex-literal = "0.3"
+
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", default-features = false }
```

Comment:
```suggestion
```
Duplicated dependency, is already set as non dev dependency

Reply:
removed